macOS/DuckDuckGo/Localizable.xcstrings

```diff
@@ -57638,7 +57638,7 @@
     },
     "preferences.always-on" : {
       "comment" : "Status indicator of a browser privacy protection feature.",
-      "extractionState" : "extracted_with_value",
+      "extractionState" : "stale",
```

Comment:
Should this string be removed?

Reply:
Stale is for ones that are no longer used. In this case of `preferences.always-on` it has moved to a separate package and is localised there.